packages/playground-ui/src/index.css

```
@@ -307,7 +307,10 @@ html.light {
   --shadow-inner: inset 0 2px 4px oklch(0 0 0 / 0.3);
   --shadow-card: 0 2px 8px oklch(0 0 0 / 0.4);
   --shadow-elevated: 0 8px 24px oklch(0 0 0 / 0.5);
-  --shadow-dialog: 0 16px 48px oklch(0 0 0 / 0.6);
+  --shadow-dialog:
+    0 0 0 1px oklch(0 0 0 / 0.04),
```

Stylelint (17.9.0)

[error] Expected "0.04" to be "4%" (alpha-value-notation)

(alpha-value-notation)